core/camel-support/src/main/java/org/apache/camel/support/task/BackgroundTask.java

```diff
@@ -155,7 +155,7 @@ private boolean waitForTaskCompletion(CountDownLatch latch, ScheduledExecutorSer
                 if (!latch.await(budget.maxDuration(), TimeUnit.MILLISECONDS)) {
                     LOG.debug("Timeout out waiting for the completion of the task");
                 } else {
-                    LOG.info("The task is complete after iterations and the code is ready to continue");
+                    LOG.info("The task has finished the execution is ready to continue");
```

Comment:
Use DEBUG logging level as we should not be noisy during processing exchanges in Camel

Reply:
Sure thing!

Reply:
Done, marking as resolved, thanks!